src/main/java/eu/dissco/core/translator/terms/License.java

```diff
@@ -10,28 +9,24 @@ public class License extends Term {
 
   public static final String TERM = "dcterms:license";
   private final List<String> dwcaTerms = List.of(TERM, "dc:license");
+  private final List<String> abcdUnitTerms = List.of("abcd:iprstatements/licenses/license/0/uri",
+      "abcd:iprstatements/licenses/license/0/text");
+  private final List<String> abcdMetaTerms = List.of(
+      "abcd:metadata/iprstatements/licenses/license/0/uri",
+      "abcd:metadata/iprstatements/licenses/license/0/text");
 
   @Override
   public String retrieveFromDWCA(JsonNode unit) {
-    return super.searchJsonForTerm(unit, dwcaTerms);
+    return super.searchJsonForStringTerm(unit, dwcaTerms);
   }
 
   @Override
-  public String retrieveFromABCD(DataSet datasets) {
-    if (datasets != null && datasets.getMetadata() != null
-        && datasets.getMetadata().getIPRStatements() != null
-        && datasets.getMetadata().getIPRStatements().getLicenses() != null
-        && !datasets.getMetadata().getIPRStatements().getLicenses().getLicense().isEmpty()) {
-      var license = datasets.getMetadata().getIPRStatements().getLicenses().getLicense().get(0)
-          .getURI();
-      if (license == null) {
-        license = datasets.getMetadata().getIPRStatements().getLicenses().getLicense().get(0)
-            .getText();
-      }
-      return license;
+  public String retrieveFromABCD(JsonNode dataset, JsonNode unit) {
+    var license = searchJsonForStringTerm(unit, abcdUnitTerms);
+    if (license == null) {
+      license = searchJsonForStringTerm(dataset, abcdMetaTerms);
     }
-    log.info("Unable to get license for dataset");
```

Comment:
Is it possible there's nothing found when the meta terms are searched as well? Maybe this log should stay

Reply:
Not sure about this. If we do it, we might do it everywhere. Lots of times the license is missing so it gives a lot of logging. We could put it on debug but then we should do it with all the terms.